dockernet/config.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -eu
+DOCKERNET_HOME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+STATE=$DOCKERNET_HOME/state
+LOGS=$DOCKERNET_HOME/logs
+SRC=$DOCKERNET_HOME/src
+PEER_PORT=26656
+DOCKER_COMPOSE="docker-compose -f $DOCKERNET_HOME/docker-compose.yml"
+
+# Logs
+SIMD_LOGS=$LOGS/simd.log
+TX_LOGS=$DOCKERNET_HOME/logs/tx.log
+KEYS_LOGS=$DOCKERNET_HOME/logs/keys.log
+
+# COIN TYPES
+# Coin types can be found at https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+COSMOS_COIN_TYPE=118
+ETH_COIN_TYPE=60
+TERRA_COIN_TYPE=330
+
+# CHAIN PARAMS
+BLOCK_TIME='1s'
+SIMD_HOUR_EPOCH_DURATION="90s"
+SIMD_DAY_EPOCH_DURATION="140s"
+SIMD_EPOCH_EPOCH_DURATION="35s"
+SIMD_MINT_EPOCH_DURATION="20s"
+UNBONDING_TIME="240s"
+MAX_DEPOSIT_PERIOD="30s"
+VOTING_PERIOD="30s"
```

Comment:
Can we set these as the block settings? (Seems that right now they are not being used)

Reply:
I removed it!